protected/humhub/modules/admin/messages/ru/base.php

```diff
@@ -1,79 +1,82 @@
+  'Invalid user state: {state}' => 'Недопустимый статус пользователя: {state}',
+  'Invite by email' => 'Пригласить с помощью электронной почты',
+  'Invite by link' => 'Пригласить с помощью специальной ссылки',
+  'Invited by' => 'Приглашен(ы)',
+  'Keep your system up-to-date and benefit from the latest improvements.' => 'Поддерживайте свою систему в актуальном состоянии и пользуйтесь недавними улучшениями.',
+  'Logging' => 'Логирование',
+  'Logs' => 'Логи',
+  'Mailing' => 'Почта',
+  'Modules' => 'Модули',
+  'No modules installed yet. Install some to enhance the functionality!' => 'Ни один из модулей не установлен. Установите необходимые модули для расширения функциональности!',
+  'Notifications' => 'Уведомления',
+  'OEmbed' => 'oEmbed',
+  'OEmbed providers' => 'Провайдеры oEmbed',
+  'Open documentation' => 'Открыть документацию',
+  'Pending user registrations' => 'Пользователи ожидающие подтверждения регистрации',
+  'People' => 'Люди',
+  'Permissions' => 'Права',
+  'Proxy' => 'Прокси',
+  'Resend to all' => 'Повторно отправить всем',
+  'Resend to selected rows' => 'Повторно отправить выделенным',
+  'Self test' => 'Самопроверка',
+  'Set as default' => 'Установить по умолчанию',
+  'Settings' => 'Настройки',
+  'Show in Marketplace' => 'Посмотреть в Маркете',
+  'Sign up' => 'Зарегистрироваться',
+  'Spaces' => 'Сообщества',
+  'Statistics' => 'Статистика',
+  'The cron job for the background jobs (queue) does not seem to work properly.' => 'Расписание cron для фоновых задач (queue) не работает должным образом.',
+  'The cron job for the regular tasks (cron) does not seem to work properly.' => 'Расписание cron для обычных задач (cron) не работает должным образом.',
+  'The mobile app push service is not available. Please install and configure the "Push Notifications" module.' => 'Служба push-уведомлений мобильного приложения недоступна. Установите и настройте модуль «Push Уведомления» («Push Notifications»).',
+  'This overview shows you all installed modules and allows you to enable, disable, configure and of course uninstall them. To discover new modules, take a look into our Marketplace. Please note that deactivating or uninstalling a module will result in the loss of any content that was created with that module.' => 'Этот раздел показывает все установленные модули и позволяет включать, выключать, настраивать и, конечно же, удалять их. Чтобы ознакомиться с новыми модулями, загляните в наш Маркет. Обратите внимание, что деактивация или удаление модуля приведёт к потере любого содержимого, созданного с помощью этого модуля.',
+  'Topics' => 'Темы',
+  'Uninstall' => 'Удалить',
+  'Updates available for {count} of your modules' => 'Доступны обновления для ваших модулей в количестве: {count}',
+  'User not found!' => 'Пользователь не найден!',
+  'Userprofiles' => 'Профили пользователей',
+  'Users' => 'Пользователи',
+  'Version' => 'Версия',
+  'Visit Marketplace' => 'Посетить Маркет',
```

Comment:
I see several good options here: **Маркет, Маркетплейс, Каталог плагинов**.

[yurabakhtin](https://github.com/yurabakhtin), write please, why you are in favor of the option with transliteration of the term?

Reply:
@yurabakhtin, thank you for discussion, I know about current usecases for `Маркетплейс`. What confused me was that this definition is closely associated with shopping on a giant platform that has absorbed the functions of an online store, a marketplace, and a platform for reselling goods from China and even `Рынок` because you can buy some sort of food too.

> The "Маркетплейс" is related more to digital goods.
Very good argument, and I agree with it!

Finallt, what do you think about `Каталог плагинов` option?

Reply:
@hide-me I am not sure exactly why the page was named as "Marketplace", maybe because some plugins have a price and can be bought there.
I don't think the phrase `Каталог плагинов` tells what data the page really has.

The `Каталог плагинов` may be related to the Module List from Administration, and it may be called as `Каталог модулей`:
![module-list](https://github.com/user-attachments/assets/272a07ac-e9fc-45d3-9875-88cbf5adbc27)

But only "Marketplace" has modules with prices:
(I guess it is why the page "Marketplace" was specially extracted from the Administration Module List and named as it is now)
![marketplace](https://github.com/user-attachments/assets/cc7604e8-b29e-443c-aa67-952c56016913)

Reply:
@yurabakhtin you are convinced me, I will change this term everywhere to `Маркетплейс`.
Thanks again for the constructive dialogue 🙏🏻!